docs/book/v1/how-tos/create-pages.md

```diff
@@ -2,16 +2,32 @@
 
 ## The `action` function
 
-The first step is to add the new pages in `src/Page/src/Controller/PageController.php`.
-This means adding an `Action` function for each page, as seen below.
+The first step is to add the new pages in `config/Autoload/local.php`.
+This means adding a `Route` for each page, as seen below.
 
 ```php
```

Comment:
Remove this PHP code because the `routes` array is already there.
See the [current version](https://github.com/dotkernel/light/blob/1.0/config/autoload/local.php.dist#L18) of the file.